src/it/java/teammates/it/storage/sqlsearch/AccountRequestSearchIT.java

```diff
@@ -164,4 +165,26 @@ private static void verifySearchResults(List<AccountRequest> actual,
         AssertHelper.assertSameContentIgnoreOrder(Arrays.asList(expected), actual);
     }
 
+    @Test
+    public void testSqlInjectionSearchAccountRequestsInWholeSystem() throws Exception {
+        ______TS("SQL Injection test in searchAccountRequestsInWholeSystem");
+
+        if (!TestProperties.isSearchServiceActive()) { 
+            return; 
+        } 
+
+        AccountRequest accountRequest = new AccountRequest("test@gmail.com", "name", "institute");
+        accountRequestsDb.createAccountRequest(accountRequest);
+
+        String searchInjection = "institute'; DROP TABLE account_requests; --";
+        List<AccountRequest> actualInjection = accountRequestsDb.searchAccountRequestsInWholeSystem(searchInjection);
+        assertEquals(0, actualInjection.size());
```

Comment:
I think the component test is failing here where 0 is expected whereas actual is 11, maybe you could check why is it returning such a value.

Reply:
yes i am trying to fix it. Thanks  for the feedback